src/plugins/discover/public/application/angular/doc_table/components/row_headers.test.js

```diff
@@ -0,0 +1,485 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import angular from 'angular';
+import 'angular-mocks';
+import 'angular-sanitize';
+import 'angular-route';
+import _ from 'lodash';
+import sinon from 'sinon';
+import { getFakeRow, getFakeRowVals } from 'fixtures/fake_row';
+import $ from 'jquery';
+import FixturesStubbedLogstashIndexPatternProvider from 'fixtures/stubbed_logstash_index_pattern';
+import { setScopedHistory, setServices, setDocViewsRegistry } from '../../../../kibana_services';
+import { coreMock } from '../../../../../../../core/public/mocks';
+import { dataPluginMock } from '../../../../../../data/public/mocks';
+import { navigationPluginMock } from '../../../../../../navigation/public/mocks';
+import { getInnerAngularModule } from '../../../../get_inner_angular';
+import { createBrowserHistory } from 'history';
+
+describe('Doc Table', () => {
+  const core = coreMock.createStart();
+  const dataMock = dataPluginMock.createStartContract();
+  let $parentScope;
+  let $scope;
+  let $elementScope;
+  let timeout;
+  let registry = [];
+
+  // Stub out a minimal mapping of 4 fields
+  let mapping;
+
+  let fakeRowVals;
+  let stubFieldFormatConverter;
+  beforeAll(() => setScopedHistory(createBrowserHistory()));
+  beforeEach(() => {
+    angular.element.prototype.slice = jest.fn(function (index) {
+      return $(this).slice(index);
+    });
+    angular.element.prototype.filter = jest.fn(function (condition) {
+      return $(this).filter(condition);
```

Comment:
oh... my eyes =) But I also don't know better solution to do it without loading jQuery for JEST

Reply:
@alexwizp for your eyes: jQuery-mock-hidding-👓